Makefile

```diff
@@ -58,13 +75,21 @@ really-clean: clean
 deploy:
 	@./scripts/deploy.sh ${BROKER_IMAGE}:${TAG} ${REGISTRY} ${ORG}
 
-test:
-	go test ./pkg/...
-
 cleanup-ci:
 	./scripts/broker-ci/cleanup-ci.sh
 
-ci:
+ci: fmtcheck vet lint build test ## Run CI (fmtcheck, vet, lint, build, test)
```

Comment:
I admit that I wasn't super keen on adding all of those dependencies to `make ci` and I think that creating a `make travis` is a good alternative. I'll make that change once I get travis to function with my changes.